daprdocs/content/en/reference/components-reference/supported-bindings/graghql.md

```diff
@@ -0,0 +1,72 @@
+---
+type: docs
+title: "GraphQL binding spec"
+linkTitle: "GraphQL"
+description: "Detailed documentation on the GraphQL binding component"
+aliases:
+  - "/operations/components/setup-bindings/supported-bindings/graphql/"
+---
+
+## Component format
+
+To setup GraphQL binding create a component of type `bindings.graphql`. See [this guide]({{< ref "howto-bindings.md#1-create-a-binding" >}}) on how to create and apply a binding configuration.
+
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: example.bindings.graphql
+spec:
+  type: bindings.graphql
+  version: v1
+  metadata:
+    - name: endpoint
+      value:  #required    
+    - name: header:xyz # e.g. header:x-hasura-access-key
```

Comment:
Should the name be `header` and the value be `xyz`? Why is value currently blank>

Reply:
No, please see https://github.com/dapr/components-contrib/pull/844#issuecomment-861009315

Reply:
Oh perfect, thanks for the clarification!